demo/addons/godotopenxrvendors/plugin.gdextension

```diff
@@ -1,7 +1,7 @@
 [configuration]
 
 entry_symbol = "plugin_library_init"
-compatibility_minimum = "4.3"
+compatibility_minimum = "4.5"
```

Comment:
If this depends on Godot 4.5, we shouldn't merge before the godot_openxr_vendors 4.x release, since that's targeting Godot 4.4+